exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
```

Comment:
Honestly, I just hadn't been aware of that file's existence and hadn't seen any documentation on it (though, admittedly, I'd just targeted briefly the pieces of documentation that seemed relevant to what I was doing since it's somewhat spread out (to me) and the elements present in other exercises seemed straightforward).

I got the base for the test file from the Hamming exercise and just edited the parts I understood (and allowed for things to pass when I ran the tests).

In short, no reason; I'll add it in.